Products/ZenCollector/utils/workers.py

```diff
@@ -130,7 +130,7 @@ def _shutdownWorkersNicely(self):
         """
         no_problem_children = True
         for worker in self._workers.values():
-            log.info("Stopping worker %s..." % worker)
+            log.info("Stopping worker %s (%s)" % (worker.pid, worker.name) )
```

Comment:
This should be written in this form:
`log.info("My message %s %s", var1, var2)`
instead of the `%` form.

This is OK because logging can delay the interpolation until later when it's determine that it will actually write the message to the log file.